### PR TITLE
Display Docker Tag in footer if the PIHOLE_DOCKER_TAG environment variable is detected

### PR DESCRIPTION
**Pull request by PromoFaux**
**By submitting this pull request, I confirm the following:**

- [x] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md), as well as this entire template.
- [x] I have made only one major change in my proposed changes.
- [x] I have commented my proposed changes within the code.
- [x] I have tested my proposed changes, and have included unit tests where possible.
- [x] I am willing to help maintain this change if there are issues with it later.
- [x] I give this submission freely and claim no ownership.
- [x] It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
- [x] I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))
- [x] I have Signed Off all commits. ([`git commit --signoff`](https://git-scm.com/docs/git-commit#git-commit---signoff))

---

**What does this PR aim to accomplish?:**

This is currently hacked into the docker container, which `sed`s it into footer.php. This is a cleaner way of doing it as it keeps the internal repo on the docker side of things in line with what is actually released.

`PIHOLE_DOCKER_TAG=dev`:

image

`PIHOLE_DOCKER_TAG` not set:

image

Companion to: pi-hole/docker-pi-hole/pull/930

### CODE REVIEW

**Review by DL6ER — APPROVED** [review]

File: scripts/pi-hole/php/footer.php [hunk 1]
```diff
@@ -49,6 +49,7 @@
     $coreVersionStr = $core_current . (isset($core_commit) ? " (" . $core_branch . ", " . $core_commit . ")" : "");
     $ftlVersionStr = $FTL_current . (isset($FTL_commit) ? " (" . $FTL_branch . ", " . $FTL_commit . ")" : "");
     $webVersionStr = $web_current . (isset($web_commit) ? " (" . $web_branch . ", " . $web_commit . ")" : "");
+    $dockerTag = getenv('PIHOLE_DOCKER_TAG') == true ? getenv('PIHOLE_DOCKER_TAG') : "";
```
Comment (DL6ER) [inline scripts/pi-hole/php/footer.php]: On a second look, this seems odd. The function is [known to return](https://www.php.net/manual/en/function.getenv.php) either `string` or `false` but never `true` ? I guess we're using a side-effect of `== true` here?

```suggestion
    $dockerTag = getenv('PIHOLE_DOCKER_TAG');
```

File: scripts/pi-hole/php/footer.php [hunk 2]
```diff
@@ -70,12 +71,14 @@
             <div class="col-xs-12 col-sm-8 col-md-6">
                 <?php if (isset($core_commit) || isset($web_commit) || isset($FTL_commit)) { ?>
                 <ul class="list-unstyled">
+                    <?php if($dockerTag) { ?> <li><strong>Docker Tag</strong> <?php echo $dockerTag; ?></li> <?php } ?>
```
Comment (DL6ER) [inline scripts/pi-hole/php/footer.php:74]:
```suggestion
                    <?php if($dockerTag !== false) { ?> <li><strong>Docker Tag</strong> <?php echo $dockerTag; ?></li> <?php } ?>
```

File: scripts/pi-hole/php/footer.php [hunk 3]
```diff
@@ -70,12 +71,14 @@
...
                     <li><strong>Pi-hole</strong> <?php echo $coreVersionStr; ?></li>
                     <li><strong>FTL</strong> <?php echo $ftlVersionStr; ?></li>
                     <li><strong>Web Interface</strong> <?php echo $webVersionStr; ?></li>
                 </ul>
                 <?php } else { ?>
                 <ul class="list-inline">
+                    <?php if($dockerTag) { ?> <li><strong>Docker Tag</strong> <?php echo $dockerTag; ?></li> <?php } ?>
```
Comment (DL6ER) [inline scripts/pi-hole/php/footer.php:81]:
```suggestion
                    <?php if($dockerTag !== false) { ?> <li><strong>Docker Tag</strong> <?php echo $dockerTag; ?></li> <?php } ?>
```

**Review by DL6ER — CHANGES_REQUESTED** [review]

File: scripts/pi-hole/php/footer.php (hunk 1 above)
Comment (PromoFaux) [reply scripts/pi-hole/php/footer.php]: Cool, I just stackoverflowed the solution :) Let me test this...

File: scripts/pi-hole/php/footer.php (hunk 2 above)
Comment (PromoFaux) [reply scripts/pi-hole/php/footer.php:74]: These changes are unnecessary - tested by both not declaring `$dockerTag` at all, and also setting `$dockerTag="";`. Also, to use it as `if ($dockerTag)` reflects the lines below (e.g `if ($FTL_Update)`)

**Comment by PromoFaux** [comment]
(Squash merge please once approved)

**Review by DL6ER — APPROVED** [review]

**Pull request merged**